_posts/en/newsletters/2021-03-31-newsletter.md

```diff
@@ -0,0 +1,142 @@
+  100 million satoshis between Alice and Bob, the paper assumes all of
+  the following states are equally as likely for that channel, and that
+  the same holds true for every other channel on the network:
+
+    | Alice | Bob |
+    | 0 sat | 100,000,000 sat |
+    | 1 sat | 99,999,999 sat |
+    | ... | ...|
+    | 100,000,000 sat | 0 sat |
+
+   Making this assumption allows the authors to draw conclusions about
+   the probability that a payment will succeed based on its amount and how
+   many hops (channels) it needs to traverse.  This allows them to
+   prove the benefit of several known heuristics---such as keeping paths
+   short and using [multipath payments][topic multipath payments] to break
+   larger payments into smaller payments (under certain other
+   assumptions).  They also use the model to evaluate new proposals,
+   such as allowing [Just-In-Time (JIT) rebalancing][topic jit routing]
+   via [bolts #780][].
+
+   The paper uses its conclusions to provide a routing algorithm that it
+   claims can reduce payment retry attempts by 20% compared to their
+   simplification of existing routing algorithms.  The new algorithm
+   prefers routes with a higher computed probability of success, whereas
+   existing algorithms use a heuristic approach.  Combined with JIT
+   rebalancing, they estimate a 48% improvement.  Given that each retry
+   usually requires several seconds, and could take much longer in some
+   cases, this could provide a significant benefit to the network.
+   The algorithm was tested against several example networks, including
+   one drawn from a snapshot of almost 1,000 live channels.
+
+   The paper deliberately does not take routing fees into consideration,
+   and most mailing list discussion about the topic focused on how to
+   use the results while still ensuring users don't pay an excessive
+   amount of fees.
+
+## Selected Q&A from Bitcoin StackExchange
+
+*[Bitcoin StackExchange][bitcoin.se] is one of the first places Optech
+contributors look for answers to their questions---or when we have a
+few spare moments to help curious or confused users.  In
+this monthly feature, we highlight some of the top-voted questions and
+answers posted since our last update.*
+
+{% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aanswer -->{% endcomment %}
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
+
+FIXME:bitschmidty
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [HWI 2.0.1][] is a bugfix release that addresses minor issues with Trezor
+  T passphrase entry and keyboard shortcuts in the `hwi-qt` user
+  interface.
+
+- [C-Lightning 0.10.0-rc1][C-Lightning 0.10.0] is a release candidate
+  for the next major version of this LN node software.
+
```

Comment:
Just released, if interesting:
- Specter desktop 1.3.0 https://github.com/cryptoadvance/specter-desktop/releases/tag/v1.3.0

Reply:
this one might be important 

https://github.com/btcpayserver/btcpayserver/releases/tag/v1.0.7.1

"This is a security release that patches one critical and several low-impact vulnerabilities that affected BTCPay Server versions 1.0.7.0 and older."

Reply:
@jonatack indeed!  Thanks!